extensions/oplogPopulator/modules/Allocator.js

```diff
@@ -60,7 +55,16 @@ class Allocator {
      * listening to the bucket
      */
     has(bucket) {
-        return this._bucketsToConnectors.has(bucket);
+        return this._connectorsManager.connectors.some(conn => conn.bucketsInUse.has(bucket));
+    }
+
+    /**
+     * Get connector assigned to a bucket
+     * @param {string} bucket bucket name
+     * @returns {Connector | undefined} connector
+     */
+    get(bucket) {
+        return this._connectorsManager.connectors.find(conn => conn.bucketsInUse.has(bucket));
```

Comment:
same, not efficient

Reply:
Closing & keeping only the discussion above.